crates/rspack_core/src/exports_info.rs

```diff
@@ -1501,19 +1507,9 @@ impl ExportInfo {
     }
   }
 
-  fn get_max_target_readonly(&self) -> &HashMap<Option<DependencyId>, ExportInfoTargetValue> {
-    assert!(self.max_target_is_set);
-    &self.max_target
-  }
-
-  fn get_max_target(&mut self) -> &HashMap<Option<DependencyId>, ExportInfoTargetValue> {
-    if self.max_target_is_set {
-      return &self.max_target;
-    }
+  fn _get_max_target(&self) -> HashMap<Option<DependencyId>, ExportInfoTargetValue> {
     if self.target.len() <= 1 {
-      self.max_target_is_set = true;
```

Comment:
why remove this set flag?

Reply:
> why remove this set flag?

This flag does not been removed, you can see it [here](https://github.com/web-infra-dev/rspack/blob/refactor/immutable-module-graph/crates/rspack_core/src/exports_info.rs#L1542), it is set in get_max_target() method, and the  _get_max_target() method is just used to generate the max target and will not modify the exports info and module graph.